### DISCUSSION TITLE
pytest test command line option to run only unexecuted tests in the suite

### DISCUSSION THREAD

**Discussion by harshagv12** [pytest-dev/pytest#8184] [Q&A]
We are running sanity test which have 10 tests in the sanity suite. In pytest command option using -x to stop test execution any failure in the sanity suite. Now I want to execute remaining tests in the sanity suite.

Example: In sanity, 6 test is failed and existed from execution. Now I want to execute the remaining tests in sanity 7 to 10,
His there any option to execute in pytest

**Comment by RonnyPfannschmidt** [comment]
that mode of operation is not implemented at all

**Reply by RonnyPfannschmidt** [reply]
i would strongly recommend to simply split the execution, first the sanity suite with no maxfail, then the rest of the suite however you like

**Reply by harshagv12** [reply]
Thanks for reply. But our structure of execution, we cannot split sanity suite with maxfail.  if we have option in pytest to run unexecuted tests it will help us.

**Reply by RonnyPfannschmidt** [reply]
its best to implement this as pytest-plugin first
